legacy/src/api/merchants/merchants.md

```diff
@@ -271,6 +271,59 @@ Returns a [paginated][] list of Merchants attached to an Account.
 }
 {% endjson %}
 
+### Search Merchants **EXPERIMENTAL**
+
+Returns a [paginated][] list of Merchants with [AssetType][] information
+
+{% reqspec %}
+  GET '/api/merchants/search'
+  auth 'api-key'
+  query_param 'origin', '123.42,32.22'
+  query_param 'distance', '100'
+  query_param 'name', 'Store'
+  query_param 'pageKey' , '2'
+  query_param 'paginationLimit', '1'
```

Comment:
Can you change pageinationLimit to something realistic? like 20 or something else